source/2019-03-30-the-ember-times-issue-91.md

```diff
@@ -0,0 +1,138 @@
+---
+title: The Ember Times - Issue No. 91
+author: Anne-Greeth van Herwijnen, Chris Ng, Jessica Jordan, Bradley Leftley, Kenneth Larsen, the crowd
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
+alias : "blog/2019/03/30-the-ember-times-issue-91.html"
+responsive: true
+---
+
+හෙලෝ Emberistas! 🐹
+
+<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+<!--alex ignore hangman-hangwoman-->
+Read some RFCs around **Record Data Errors** 📇 and extending `EmberObject` 💡, learn how to build Hangman with **Ember Octane** 🎬 and about the **addon modules** RFC, check out the "e"mpressive EmberWeekend podcast with **@mansona**, ...
+
+
+...and read the exclusive interview with Ember contributor **@kategengler**.
+
+---
+
+## [Configuring Addon Modules in Module Unification Layout](https://github.com/ppcano/rfcs/blob/addon-module-config/text/0000-addon-module-config.md)
+
+A new RFC is out which proposes to provide an **API for addons** to configure their **module types** and collections for **Module Unification apps**. The reason for this is that at this moment Classic addons that require apps to locate classes into a non-default `app/` folder do not work "out of the box" in a Module Unification application.
```

Comment:
To indicate that Module Unification got officially rescoped shortly after the publication of the aforementioned RFC, we might want to mention something like:

```suggestion
A new RFC is out which proposes to provide an **API for addons** to configure their **module types** and collections for **Module Unification apps**. The reason for this is that at this moment Classic addons that require apps to locate classes into a non-default `app/` folder do not work "out of the box" in an application built according to the [yet unreleased API design proposed in the original Module Unification RFC](https://github.com/emberjs/rfcs/pull/143).
```